enterprise/api/utils.py

```diff
@@ -29,3 +33,23 @@ def get_enterprise_customer_from_catalog_id(catalog_id):
         return str(EnterpriseCustomerCatalog.objects.get(pk=catalog_id).enterprise_customer.uuid)
     except EnterpriseCustomerCatalog.DoesNotExist:
         return None
+
+
+def get_ent_cust_from_report_config_uuid(uuid):
```

Comment:
nit: I think it's easier to read if we spell out enterprise_customer instead of ent_cust.

Reply:
It will not pass the quality lint check due to length